libs/galley-types/test/unit/Test/Galley/Types.hs

```diff
@@ -54,7 +55,35 @@ tests =
         -- this test, and force future develpers to consider what permissions they want to set.
         assertBool "all covered" (all (roleHasPerm RoleExternalPartner) (ViewTeamFeature <$> [minBound ..])),
       testRoundTrip @FeatureFlags,
-      testRoundTrip @GuardLegalholdPolicyConflicts
+      testRoundTrip @GuardLegalholdPolicyConflicts,
+      testGroup
+        "permissionsRole, rolePermissions"
+        [ testCase "'Role' maps to expected permissions" $ do
+            assertEqual "role type changed" [minBound ..] [RoleOwner, RoleAdmin, RoleMember, RoleExternalPartner]
+            assertEqual "owner" (permissionsRole =<< newPermissions (intToPerms 8191) (intToPerms 8191)) (Just RoleOwner)
```

Comment:
it might be nicer to use hexadecimal, octal, or binary (does haskell have binary?) notation for these constants. I cant map base-10 to bitmaps in my head

Reply:
me neither, but this is just protection against regression, and unrelated to the changes in this PR.

do you think it would be beneficial (*and* worth the time) to have it in hex or binary?  i personally don't think i could extract more useful information from that.

Reply:
if you really want this i'm happy to do it in a separate PR that you can review any time in the future, would that be ok?

Reply:
ah these are from the counter example you found in the wild? got it